src/controllers/ajax/AdminAjaxController.php

```diff
@@ -410,7 +425,55 @@ protected function getActions(): array<string> {
         Control::backupDb();
         return Utils::ok_response('Success', 'admin');
       case 'export_game':
-        Control::exportGame();
+        await Control::exportGame();
+        return Utils::ok_response('Success', 'admin');
+      case 'export_teams':
+        await Control::exportTeams();
+        return Utils::ok_response('Success', 'admin');
+      case 'export_logos':
+        await Control::exportLogos();
+        return Utils::ok_response('Success', 'admin');
+      case 'export_levels':
+        await Control::exportLevels();
+        return Utils::ok_response('Success', 'admin');
+      case 'export_categories':
+        await Control::exportCategories();
+        return Utils::ok_response('Success', 'admin');
+      case 'import_game':
+        $result = await Control::importGame();
+        if ($result) {
+          return Utils::ok_response('Success', 'admin');
+        } else {
```

Comment:
It's usually cleaner to just remove the `else` if the conditional is going to return.

```
if ($result) {
    return value;
}

return value2;
```

Alternatively you could make these a one-liner with ternary operators, though I don't think it'd be cleaner.
```
return $result ? Utils::ok_response('Success', 'admin') : Utils::error_response('Error importing', 'admin');
```